src/app/pages/storage/volumes/volumes-list/volumes-list.component.html

```diff
@@ -2,6 +2,14 @@
   <ix-page-title-header>
     <ix-volumes-list-controls [entity]="this" [conf]="conf"></ix-volumes-list-controls>
   </ix-page-title-header>
+
+  <mat-card class="deprecated">
+    <h4 mat-card-title>
+      <mat-icon>warning</mat-icon>
+      {{ 'This page is deprecated and will be removed in Bluefin Release. If you find yourself still needing to use old Storage pages, please' | translate }}
+      <button mat-button (click)="onFileTicketPressed()">{{ 'File Ticket' | translate }}</button>
```

Comment:
Instead of making this a button, let's make a link similar to other links in the redesign or finish the sentence to say `, please file a ticket.` It doesn't look nice if the screen is short.
![image](https://user-images.githubusercontent.com/24204846/181862576-08f007e4-c950-4be5-aee5-c12086b6cd1a.png)

Reply:
Agreed. Looks weird when 2 lines.